tests/integration/backup_tests/test_backups.py

```diff
@@ -84,7 +89,7 @@ async def test_blocked_incorrect_creds(ops_test: OpsTest) -> None:
     await ops_test.model.wait_for_idle(apps=[S3_APP_NAME], status="active")
 
     await wait_for_mongodb_units_blocked(
-        ops_test, db_app_name, status="s3 credentials are incorrect.", timeout=300
+        ops_test, db_app_name, status="s3 credentials are incorrect.", timeout=600
```

Comment:
hmmm, I understand that a longer time out is needed, but also 10minutes is a long time and our tests are already taking a long time to run. 

Can we manually configure juju to have a shorter update status for these tests? This way they go into their expected states more quickly?

I am not sure `ops_test.fast_forward` will work since we are using subprocess to get the juju status. Maybe adding this in `wait_for_mongodb_units_blocked `:
```
await ops_test.model.set_config({"update-status-hook-interval": "15s"})
# check status
await ops_test.model.set_config({"update-status-hook-interval": "5m"})
```